test/notAllPass.js

```diff
@@ -9,21 +10,21 @@ describe('notAllPass', function() {
 
   it('reports whether all predicates are satisfied by a given value', function() {
     const ok = RA.notAllPass([odd, lt20, gt5]);
-    eq(ok(7), false); // all ps succeed
-    eq(ok(10), true); // p1 fails
-    eq(ok(21), true); // p2 fails
-    eq(ok(3), true); // p3 fails
-    eq(ok(22), true); // p1 and p2 fails
-    eq(ok(4), true); // p1 and p3 fails
+    assert.isFalse(ok(7)); // all ps succeed
+    assert.isTrue(ok(10)); // p1 fails
+    assert.isTrue(ok(21)); // p2 fails
+    assert.isTrue(ok(3)); // p3 fails
+    assert.isTrue(ok(22)); // p1 and p2 fails
+    assert.isTrue(ok(4)); // p1 and p3 fails
   });
 
   it('returns false on empty predicate list', function() {
-    eq(RA.notAllPass([])(3), false);
+    assert.isFalse(RA.notAllPass([])(3));
   });
 
   it('returns a curried function whose arity matches that of the highest-arity predicate', function() {
-    eq(RA.notAllPass([odd, gt5, plusEq]).length, 4);
-    eq(RA.notAllPass([odd, gt5, plusEq])(9, 9, 9, 10), true);
-    eq(RA.notAllPass([odd, gt5, plusEq])(9)(9)(9)(10), true);
+    assert.strictEqual(RA.notAllPass([odd, gt5, plusEq]).length, 4);
```

Comment:
Checkout the [lengthOf](https://www.chaijs.com/api/assert/#method_lengthof) method of chai. More idiomatic to use it here IMHO